docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md

```diff
@@ -36,19 +37,54 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
    --data-raw "{
        \"targets\": {
            \"include\": {\
-               \"runtime_id\": "uuid-sdasd-sda23t-efs",
+               \"runtimeID\": "uuid-sdasd-sda23t-efs",
+               \"globalAccount\": "uuid-sdasd-sda23t-efs",
+               \"subAccount\": "uuid-sdasd-sda23t-efs",
+               \"planName\": "azure",
+               \"region\": "europewest",
             },
        },
-        \"dry\": false,
+       \"dryRun\": false
    }"
    ```
 
+>**NOTE:** Without specifying the **targets** field, the orchestration will select all the runtimes by default using the **all** target.
```

Comment:
```suggestion
>**NOTE:** By default, the orchestration will select all the runtimes using the **all** target.
```

Reply:
Will remove that sentence because implementation has changed after a review